packages/test/test-service-load/package.json

```diff
@@ -37,8 +37,10 @@
     "prettier:fix": "prettier --write . --ignore-path ../../../.prettierignore",
     "start": "node ./dist/nodeStressTest.js",
     "start:frs": "node ./dist/nodeStressTest.js --driver routerlicious --driverEndpoint frs --profile ci_frs",
+    "start:frs:large": "node ./dist/nodeStressTest.js --driver routerlicious --driverEndpoint frs --profile ci_with_large_payloads",
     "start:mini": "node ./dist/nodeStressTest.js  --profile mini",
     "start:odsp": "node ./dist/nodeStressTest.js --driver odsp",
+    "start:odsp:large": "node ./dist/nodeStressTest.js --driver odsp --profile ci_with_large_payloads",
```

Comment:
can we avoid adding extra modes here? And instead use main mode, but add capability to test various configurations in main run?

Reply:
That's the end goal. I want this to be separated for now.